src/main/scala/com/amazon/deequ/profiles/ColumnProfiler.scala

```diff
@@ -133,7 +139,11 @@ object ColumnProfiler {
 
     val firstPassResults = analysisRunnerFirstPass.run()
 
-    val genericStatistics = extractGenericStatistics(relevantColumns, data.schema, firstPassResults)
+    val genericStatistics = extractGenericStatistics(
```

Comment:
no need to change the formatting here

Reply:
The only reason that I am changing the format here is to avoid the "line length exceeding 100" violation when building.